client/src/components/App/App.css

```diff
@@ -52,7 +52,7 @@ body {
   width: 100vw;
   height: 100vh;
   background-color: #fff;
-  overflow: hidden;
+  overflow: auto;
```

Comment:
I think the only change we need for the PR is to remove this `overflow` line. Will add a video of what I think this is trying to do.